actions/download_action.go

```diff
@@ -29,14 +29,21 @@ See the 'Unpack' action for more information.
 
 - compression -- optional hint for unpack allowing to use proper compression method.
 See the 'Unpack' action for more information.
+
+- sha256sum -- optional expected SHA256 sum of the downloaded file
```

Comment:
would be wroth noting that this is expected to be a hex string of 64 characters (i assume it can't use the 0x prefix but is not case-sensitive?)

Reply:
Indeed, that's not case sensitive; I've clarified the description

We could strip a leading 0x, but I personally would prefer if we stick to a single way of writing these